__playwright__/circulars/corrections.spec.ts

```diff
@@ -13,12 +13,9 @@ const loadingTestsCircular = {
   version: 3,
 }
 
-function getDateAndTimeStrings(createdOn: number) {
+function getDateTimeString(createdOn: number) {
   const date = new Date(createdOn)
-  return [
-    date.toISOString().split('T')[0],
-    `${date.getUTCHours() % 12}:${date.getUTCMinutes()}${date.getUTCHours() > 12 ? 'pm' : 'am'}`,
-  ]
+  return date.toISOString()
```

Comment:
This function is defined identically in multiple files.

Instead of defining it again here, I suggest just hard-coding the expected output for the test below.